docs/src/architecture/data-model.md

```diff
@@ -2,7 +2,7 @@
 # thin-edge Data Model
 
 The **data model** identifies all data send or received from/to **thin-edge** and its components, to interact with those.
-For all data it defines format and explains behaviour.
+For all data it defines format and explains behavior.
```

Comment:
I was under the impression that UK english is still acceptable, as they are the pioneers of the English language ;)